realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -118,10 +118,19 @@ public boolean shouldDeleteRealmIfMigrationNeeded() {
         return durability;
     }
 
-    public RealmProxyMediator getSchemaMediator() {
+    RealmProxyMediator getSchemaMediator() {
         return schemaMediator;
     }
 
+    /**
+     * Returns the unmodifiable {@link Set} of model classes that belong to this configuration.
```

Comment:
Perhaps "Returns the unmodifiable Set of model classes that make up the schema for this Realm?